offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/offer/create/CreateOfferViewModel.groovy

```diff
@@ -75,18 +75,45 @@ class CreateOfferViewModel {
     private final ProductsResourcesService productsResourcesService
     private final ProjectManagerResourceService managerResourceService
 
+    @Bindable Boolean offerCreatedSuccessfully
+
     CreateOfferViewModel(CustomerResourceService customerResourceService,
                          ProjectManagerResourceService managerResourceService,
                          ProductsResourcesService productsResourcesService) {
         this.customerResourceService = customerResourceService
         this.productsResourcesService = productsResourcesService
         this.managerResourceService = managerResourceService
 
+        offerCreatedSuccessfully = false
+        this.addPropertyChangeListener("offerCreatedSuccessfully", {
+            resetModel()
+        })
```

Comment:
do you distinguish here between true and false of the property? It seems like the reset is triggered for every change of the variable. Is that intended?

Reply:
nice spot